modules/gateway/peers.go

```diff
@@ -168,7 +168,7 @@ func (g *Gateway) Connect(addr modules.NetAddress) error {
 		return err
 	}
 	// send our version
-	if err := encoding.WriteObject(conn, build.Version); err != nil {
+	if err := encoding.WriteObject(conn, "0.3.3"); err != nil {
```

Comment:
I was actually just thinking that it would be cool to gather some version metrics, i.e. what percentage is running each version. I think I could write a test program to determine this pretty easily.